src/Form/LegacyConsumer/Commands/SetupFieldPersistance.php

```diff
@@ -19,8 +19,8 @@ function( $donationID, $donationData ) use ( $hook ) {
 				do_action( "give_fields_$hook", $fieldCollection, $donationData['give_form_id'] );
 				$fieldCollection->walk(
 					function( $field ) use ( $donationID ) {
-						if ( isset( $_POST[ 'give_' . $field->getName() ] ) ) {
-							$value = wp_strip_all_tags( $_POST[ 'give_' . $field->getName() ], true );
+						if ( isset( $_POST[ $field->getName() ] ) ) {
+							$value = wp_strip_all_tags( $_POST[ $field->getName() ], true );
```

Comment:
This fixes a bug in persisting the custom field value - the prefix was removed from the field name but not the logic.